src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1322,6 +1322,8 @@ dataset.locked.ingest.message=The tabular data files uploaded are being processe
 dataset.unlocked.ingest.message=The tabular files have been ingested.
 dataset.locked.editInProgress.message=Edit In Progress
 dataset.locked.editInProgress.message.details=Additional edits cannot be made at this time. Contact {0} if this status persists.
+dataset.locked.pidNotReserved.message=Dataset Locked
```

Comment:
I don't think we need to add another "Dataset Locked" bundle value here. There is already `dataset.locked.message`.

Reply:
I put the key in as a placeholder but just changed it to "Dataset DOI Not Reserved" in fee17726d.